TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Migration/Service/NodeTransformation.php

```diff
@@ -32,14 +35,49 @@ class NodeTransformation
     protected $transformationConjunctions = array();
 
     /**
+     * @Flow\Inject
+     * @var \TYPO3\Eel\EelEvaluatorInterface
+     */
+    protected $eelEvaluator;
+
+    /**
+     * @Flow\Inject
+     * @var NodeFactory
+     */
+    protected $nodeFactory;
+
+    /**
+     * @Flow\InjectConfiguration("transformations.eel.defaultContext")
+     * @var array
+     */
+    protected $defaultContextConfiguration;
+
+    /**
+     * @var array
+     */
+    protected $contextVariables;
+
+    /**
+     * @return void
+     */
+    public function initializeObject()
+    {
+        if ($this->eelEvaluator instanceof \TYPO3\Flow\Object\DependencyInjection\DependencyProxy) {
```

Comment:
Thanks for the explanation. Indeed switching off lazy injection should be done. I'll update the code…